flink-runtime/src/main/java/org/apache/flink/runtime/dispatcher/Dispatcher.java

```diff
@@ -169,100 +169,35 @@ public Dispatcher(
             DispatcherBootstrapFactory dispatcherBootstrapFactory,
             DispatcherServices dispatcherServices)
             throws Exception {
-        this(
-                rpcService,
-                fencingToken,
-                recoveredJobs,
-                recoveredDirtyJobs,
-                dispatcherBootstrapFactory,
-                dispatcherServices,
-                new JobManagerRunnerRegistry(16));
-    }
-
-    private Dispatcher(
```

Comment:
I wasn't sure whether it would break the Github PR view here. That's why I added the revert. I will squash the commits properly after you give your thumbs up on squashing